app/views/course/announcements/index.json.jbuilder

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+json.announcementTitle @settings.title
```

Comment:
this one is actually because for leaderboardTitle, that's how they do it
`json.leaderboardTitle @settings.title` is already existing, and therefore I just follow suit